activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -448,12 +442,18 @@ def load_seed
       #
       # ==== Examples:
       #   ActiveRecord::Tasks::DatabaseTasks.dump_schema_cache(ActiveRecord::Base.connection, "tmp/schema_dump.yaml")
-      def dump_schema_cache(conn, filename)
+      def dump_schema_cache(conn, namespace, schema_cache_path:)
```

Comment:
This change is breaking the API of a public class. I'm fine with it but we need to point in the changelog and also update the example above.

Reply:
Ah! I think we should give it a default of `nil`, that way it is compatible with the public API.